common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -1,4 +1,4 @@
-// democracylab.org/companies page --- TODO: rename controller
+:ww// democracylab.org/companies page --- TODO: rename controller
```

Comment:
```suggestion
// democracylab.org/companies page --- TODO: rename controller
```
tiny clean-up item; removing this errant comment clears a build error.